CMakeLists.txt

```
@@ -14,7 +14,12 @@ execute_process(
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
     OUTPUT_VARIABLE VERSION_STR
     OUTPUT_STRIP_TRAILING_WHITESPACE
+    RESULT_VARIABLE GITVER
 )
+if(NOT GITVER EQUAL "0")
```

Nitpick: to be consistent with the rest of the file

```suggestion

if (NOT GITVER EQUAL "0")
```